packages/perspective/src/js/perspective.parallel.js

```diff
@@ -14,7 +14,7 @@ import {worker} from "./api.js";
 import asmjs_worker from "./perspective.asmjs.js";
 import wasm_worker from "./perspective.wasm.js";
 
-import wasm from "./psp.async.wasm.js";
+import wasm_path from "./psp.async.wasm.js";

```

Comment:
Because this was also added to `perspective.wasm.js` above, this asset is inlined twice in the jupyterlab plugin